src/test/java/gov/healthit/chpl/aqa/asserts/ViewVersionPageAsserts.java

```diff
@@ -40,5 +40,26 @@ public void iSeeMergeLinkToMergeTheVersion(final String versionId) {
         ViewVersionPage.mergeVersionLink(getDriver(), versionId).isDisplayed();
         assertTrue(true);
     }
+
+    /**
+     * Assert that "At least one other Version must be selected to merge" is displayed in versions to merge section.
+     * @param versionsToMergeMessage is a section in merge versions
+     */
+    @Then("^I see \"([^\"]*)\" in versions to merge section$")
+    public void iSeeMessageInVersionsToMergeSection(final String versionsToMergeMessage) {
+        ViewVersionPage.versionsToMergeMessage(getDriver()).isDisplayed();
```

Comment:
This string isn't being passed into the page object anywhere, so even if this does work, changing the string in the feature file, or changing the message on the page itself, wouldn't cause this assert to fail, which means the test isn't right